specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyStates.json

```diff
@@ -83,9 +83,15 @@
             }
           }
         },
+        "x-ms-pageable": {
+          "nextLinkName": "@odata.nextLink"
+        },
```

Comment:
readme.md is not referencing 2018-07-01-preview tag, so this change is not really going to be effective. Either revert this json or add 2018-07-01-preview tag to reference this spec file.

Reply:
never mind, i noticed it's used for 2018-07 tag,

---
In reply to: [355047766](https://github.com/Azure/azure-rest-api-specs/pull/7909#discussion_r355047766) [](ancestors = 355047766)